includes/polling/applications/bind.inc.php

```diff
@@ -2,13 +2,11 @@
 
 use LibreNMS\RRD\RrdDefinition;
 
-echo ' bind ';
-
+$name = 'bind';
```

Comment:
Thanks, I didn't realize that was preferred as it seems kind of inconsistent in these inc files.  But I've added back in as suggested.